test/functional/collations_tests.js

```diff
@@ -633,7 +633,7 @@ describe('Collation', function() {
     }
   });
 
-  it('Fail due to no support for collation', {
+  it.skip('Fail due to no support for collation', {
```

Comment:
left a skip in here

Reply:
oops should have replied to this thread, but see below comment about why I skipped this one